src/components/Modal/ModalContent/AddMovieForm/AddMovieForm.jsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+
+export default function AddMovieForm() {
```

Comment:
how about have one common component for `AddMovieForm` and `EditMovieForm`? What are the differences there beside populated data when edit mode?